electron.gyp

```diff
@@ -436,20 +436,17 @@
           'action_name': 'atom_browserify',
           'inputs': [
             '<@(browserify_entries)',
-            # Any js file under `lib/` can be included in the preload bundle.
-            # Add all js sources as dependencies so any change to a js file will
-            # trigger a rebuild of the bundle(and consequently of js2c).
-            '<@(js_sources)',
           ],
           'outputs': [
             '<(js2c_input_dir)/preload_bundle.js',
           ],
           'action': [
             'npm',
             'run',
+            '--silent',
```

Comment:
This changes makes it so the progress bar isn't interrupted when this target runs, it previously looked like this:

```
> electron@1.4.5 build /Users/kevin/github/electron
> python ./script/build.py -c D

ninja: Entering directory `out/D'
[2/20] ACTION atom_browserify: atom_browserify_dd89504f08caf5c5c2049d99ba64c2b4

> electron@1.4.5 browserify /Users/kevin/github/electron
> browserify "lib/sandboxed_renderer/init.js" "-o" "out/D/gen/js2c/preload_bundle.js"

[20/20] STAMP Electron.app
```

It will now look like:

```
> electron@1.4.5 build /Users/kevin/github/electron
> python ./script/build.py -c D

ninja: Entering directory `out/D'
[20/20] STAMP Electron.app
```